device/src/mac/mod.rs

```diff
@@ -0,0 +1,178 @@
+use crate::{
+    radio::RadioBuffer, region, state_machines::Downlink, FcntDown, FcntUp, SendData, SessionKeys,
+};
+use generic_array::{typenum::U256, GenericArray};
```

Comment:
Just a note: it would be good to change this to use const generics instead of generic array at some point.

Reply:
Good point. Left an issue to track it #126